tests/test_utility.py

```diff
@@ -147,21 +147,19 @@ def test_2_cats_jealousy(self):
         relation_4_2.link_relationship()
 
         # when
-        relation_1_2.jealousy += 20
-        relation_3_2.jealousy += 20
-        relation_4_2.jealousy += 10
+        relation_1_2.respect += -20
```

Comment:
```suggestion
        relation_1_2.respect -= 20
```
repeat for the others